docs-sources/includes/_Configuration.md

```diff
@@ -55,6 +66,26 @@ To run your game in development, MAGE has a `developmentMode` configuration flag
 disables certain behaviors which make development a bit more convenient. If you want more granular
 control over which of these behaviors are turned on or off, you can specify them in an object.
 
+## Environment-based configuration
+
+```yaml
+server:
+    clientHost:
+      bind: MAGE_APP_BIND:string
+
+    mmrp: MAGE_APP_MMRP:bool
+```
+
+You may also add a `config/environment.yaml` file to your project: this file serves
+as a mean to connect environment variables to specific configuration entries. Environment
```

Comment:
"a means" I think
Maybe a native English speaker can confirm

Reply:
I ran this into Grammarly, and it did not return any errors or suggested any changes.